catalog/dags/data_refresh/recreate_view_data_task_factory.py

```diff
@@ -0,0 +1,81 @@
+"""
+# Refresh Materialized View Task Factory
+This file generates a Task that recreates the materialized view for a
```

Comment:
Wouldn't this be a `TaskGroup`?
```suggestion
This file generates a TaskGroup that recreates the materialized view for a
```